developer_manual/basics/backgroundjobs.rst

```diff
@@ -62,6 +62,25 @@ to pass on to the service to run the background job.
 The ``run`` function is the main thing you need to implement and where all the
 logic happens.
 
+Heavy load and time insensitive
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When the background job can impact the performance of the instance and is not time sensitive,
+e.g. when clearing old data, running training of AI models or similar things, consider flagging
+it as time insensitive in the constructor.
+
+.. code-block:: php
+
+    <?php
+
+    // Run once a day
+    parent::setInterval(24 * 3600);
```

Comment:
why parent and not `$this->`? it's a dynamic method

Reply:
just copied from the sample above